library/analytics/src/main/java/org/cru/godtools/analytics/model/OpenToolsAnalyticsActionEvent.kt

```diff
@@ -0,0 +1,10 @@
+package org.cru.godtools.analytics.model
+
+sealed class OpenToolsAnalyticsActionEvent(action: String) : AnalyticsActionEvent(action) {
+    override fun isForSystem(system: AnalyticsSystem): Boolean {
+        return system == AnalyticsSystem.APPSFLYER
+    }
+}
+
+object ToolOpened : OpenToolsAnalyticsActionEvent("tool-opened")
+object FirstToolOpened : OpenToolsAnalyticsActionEvent("first-tool-opened")
```

Comment:
let's move this class into the appropriate package within the `:ui:base-tool` module